src/utils/transactions.ts

```diff
@@ -198,11 +198,13 @@ export const stringAmountToBalance = (formAmount: string) => {
   return Balance.fromFloat(floatAmount, CurrencyType.networkToken)
 }
 
-export const getMemoBytesLeft = (
-  memo: string,
-): { numBytes: number; valid: boolean } => {
+export const getMemoBytesLeft = (memo: string, shouldEncode = false) => {
```

Comment:
generally I try to prefer not adding options as top level arguments to a function because when the function is used, it's not clear what the argument controls. For example, you'd need to look at the definition of the function to see what the `true` in `getMemoBytesLeft(memo, true)` is telling the function to do. You could maybe use an options argument instead which would be self-documenting: `getMemoBytesLeft(memo, { shouldEncode: true })`. I've also been trying to prefer React components over utility functions in the explorer because props are the equivalent of an options object although that may not be the right solution here

Reply:
Totally agree that options are a better way to go. I ended up removing the encoding from this utility function altogether. I think it's much clearer now.